packages/devextreme/js/__internal/ui/menu/m_menu.ts

```diff
@@ -275,7 +275,15 @@ class Menu extends MenuBase {
     // @ts-expect-error
     this._addCustomCssClass(this.$element());
 
-    this.setAria('role', 'menubar');
+    this._setRoleAria();
+  }
+
+  _setRoleAria(): void {
+    if (!this._isAdaptivityEnabled()) {
+      this.setAria('role', 'menubar');
+    } else {
+      this.setAria('role', undefined);
```

Comment:
There is not role menubar, when menu is not collapsed and adaptivityEnabled: true:

<img width="1094" alt="image" src="https://github.com/user-attachments/assets/24f535e3-ee66-421f-8a73-da1272f587ee">

Reply:
Also in this case we have some axe issues:

<img width="1713" alt="image" src="https://github.com/user-attachments/assets/5b4ca0ea-455f-4e4d-a3ba-24f6da5e52fc">

Reply:
Please set the menubar role only when the menu is minimized into a button. Look at _dimensionChanged method.

Reply:
[cde1139](https://github.com/DevExpress/DevExtreme/pull/28066/commits/cde113923df248a1be7a0ed54c2c2220067cc24a)